src/common/util.ts

```diff
@@ -61,3 +61,25 @@ export function convertPropertiesToFields(properties : SectionProperty[]): { fie
   extractFields(properties);
   return result;
 }
+
+export function unsetObjectKeys<T extends object>(objectTag: T, keysToRemove: string[]): string | null {
```

Comment:
Can we not do JSON stringify here? Let the other codepoints where this is called decide stringify or not